### ISSUE TITLE
Improve extensibility and maintainability for new contributors

### ISSUE THREAD

**Issue by ProfJanetDavis** [gender-bias/gender-bias-web#36]
So that future contributors can understand this code more easily, we will review our code and identify opportunities for clarification, whether by refactoring or by improved documentation - for example, in comments, or in a separate document providing an introduction for contributors.

If this additional document is created, some documentation may be moved here from the `README`. See [`gender-bias/CONTRIBUTING.md`](https://github.com/gender-bias/gender-bias/blob/master/CONTRIBUTING.md) as a model.

When you see something that needs doing, unless it's tiny, please make a new issue before starting your work!

- [x] Review entire codebase to look for things that are confusing or unclear.
- [x] Review tests and think if any are missing or redundant.
- [x] Clone/serve/test from scratch to look for things that are unclear or steps that are missing.
- [ ] Document how the front and back end are deployed? Ask @j6k4m8 if this seems helpful.

@nidhi2509 will not contribute to this issue (or contribute very minimally) so she can focus on #34.

**Comment by ProfJanetDavis** [comment]
PR #46 currently blocked on issue #47

**Comment by ProfJanetDavis** [comment]
PR #46 currently blocked on issue #47
